pkg/tools/sandbox/fake_resolver.go

```diff
@@ -55,35 +62,31 @@ func (f *FakeDNSResolver) LookupSRV(ctx context.Context, service, proto, name st
 }
 
 // LookupIPAddr lookups IP address by host
-func (f *FakeDNSResolver) LookupIPAddr(_ context.Context, host string) ([]net.IPAddr, error) {
+func (f FakeDNSResolver) LookupIPAddr(_ context.Context, host string) ([]net.IPAddr, error) {
```

Comment:
Interesting. Why do we need to make the receiver a struct here?

Reply:
No need, I'll modify it to use pointers instead.